scripts/carrot/HaplotypeCallerComparisonForCarrot.monitoring.wdl

```diff
@@ -0,0 +1,372 @@
+version 1.0
+ 
+ workflow VariantCallingCarrot {
+
+  String pipeline_version = "2.0.3"
+
+  input {
+    Boolean run_dragen_mode_variant_calling = false
+    Boolean use_spanning_event_genotyping = true
+    File calling_interval_list
+    Int haplotype_scatter_count
+    Int break_bands_at_multiples_of
+    Float? contamination = 0
+    File input_bam
+    File input_bam_index
+    File ref_fasta
+    File ref_fasta_index
+    File ref_dict
+    File? ref_str
+    String base_file_name
+    String final_vcf_base_name
+    Int agg_preemptible_tries
+    String gatk_docker = "us.gcr.io/broad-gatk/gatk:4.2.0.0"
```

Comment:
Should we have a default for this (and in the other places where 4.2.0.0 is being set as the default), since the intention is that this is the thing that's being tested?  I think it might be better to not have one so that if we accidentally (or because of a bug) don't supply a value for this, it will cause the test to fail and call our attention to the issue.